--- php.ini-recommended.orig	2011-05-30 15:51:02.411642461 +0200
+++ php.ini-recommended	2011-05-30 15:53:30.367264799 +0200
@@ -251,7 +251,7 @@
 ; and below.  This directive makes most sense if used in a per-directory
 ; or per-virtualhost web server configuration file. This directive is
 ; *NOT* affected by whether Safe Mode is turned On or Off.
-;open_basedir =
+open_basedir = /srv/http/:/home/:/tmp/:/opt/php52/pear/
 
 ; This directive allows you to disable certain functions for security reasons.
 ; It receives a comma-delimited list of function names. This directive is
@@ -522,7 +522,7 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;
 
 ; UNIX: "/path1:/path2"
-;include_path = ".:/php/includes"
+include_path = ".:/opt/php52/share/pear"
 ;
 ; Windows: "\path1;\path2"
 ;include_path = ".;c:\php\includes"
@@ -539,7 +539,7 @@
 user_dir =
 
 ; Directory in which the loadable extensions (modules) reside.
-extension_dir = "./"
+extension_dir = "/opt/php52/modules"
 
 ; Whether or not to enable the dl() function.  The dl() function does NOT work
 ; properly in multithreaded servers, such as IIS or Zeus, and is automatically
@@ -644,7 +644,7 @@
 ;
 ; For example, on Windows:
 ;
-;   extension=msql.dll
+;   extension=msql.so
 ;
 ; ... or under UNIX:
 ;
@@ -655,57 +655,51 @@
 ; extension_dir directive above.
 
 
-; Windows Extensions
-; Note that ODBC support is built in, so no dll is needed for it.
-; Note that many DLL files are located in the extensions/ (PHP 4) ext/ (PHP 5)
-; extension folders as well as the separate PECL DLL download (PHP 5).
-; Be sure to appropriately set the extension_dir directive.
-
-;extension=php_bz2.dll
-;extension=php_curl.dll
-;extension=php_dba.dll
-;extension=php_dbase.dll
-;extension=php_fdf.dll
-;extension=php_gd2.dll
-;extension=php_gettext.dll
-;extension=php_gmp.dll
-;extension=php_ifx.dll
-;extension=php_imap.dll
-;extension=php_interbase.dll
-;extension=php_ldap.dll
-;extension=php_mbstring.dll
-;extension=php_exif.dll
-;extension=php_mcrypt.dll
-;extension=php_mhash.dll
-;extension=php_mime_magic.dll
-;extension=php_ming.dll
-;extension=php_msql.dll
-;extension=php_mssql.dll
-;extension=php_mysql.dll
-;extension=php_mysqli.dll
-;extension=php_oci8.dll
-;extension=php_openssl.dll
-;extension=php_pdo.dll
-;extension=php_pdo_firebird.dll
-;extension=php_pdo_mssql.dll
-;extension=php_pdo_mysql.dll
-;extension=php_pdo_oci.dll
-;extension=php_pdo_oci8.dll
-;extension=php_pdo_odbc.dll
-;extension=php_pdo_pgsql.dll
-;extension=php_pdo_sqlite.dll
-;extension=php_pgsql.dll
-;extension=php_pspell.dll
-;extension=php_shmop.dll
-;extension=php_snmp.dll
-;extension=php_soap.dll
-;extension=php_sockets.dll
-;extension=php_sqlite.dll
-;extension=php_sybase_ct.dll
-;extension=php_tidy.dll
-;extension=php_xmlrpc.dll
-;extension=php_xsl.dll
-;extension=php_zip.dll
+;extension=php_bz2.so
+;extension=php_curl.so
+;extension=php_dba.so
+;extension=php_dbase.so
+;extension=php_fdf.so
+;extension=php_gd2.so
+;extension=php_gettext.so
+;extension=php_gmp.so
+;extension=php_ifx.so
+;extension=php_imap.so
+;extension=php_interbase.so
+;extension=php_ldap.so
+;extension=php_mbstring.so
+;extension=php_exif.so
+;extension=php_mcrypt.so
+;extension=php_mhash.so
+;extension=php_mime_magic.so
+;extension=php_ming.so
+;extension=php_msql.so
+;extension=php_mssql.so
+;extension=php_mysql.so
+;extension=php_mysqli.so
+;extension=php_oci8.so
+;extension=php_openssl.so
+;extension=php_pdo.so
+;extension=php_pdo_firebird.so
+;extension=php_pdo_mssql.so
+;extension=php_pdo_mysql.so
+;extension=php_pdo_oci.so
+;extension=php_pdo_oci8.so
+;extension=php_pdo_odbc.so
+;extension=php_pdo_pgsql.so
+;extension=php_pdo_sqlite.so
+;extension=php_pgsql.so
+;extension=php_pspell.so
+;extension=php_shmop.so
+;extension=php_snmp.so
+;extension=php_soap.so
+;extension=php_sockets.so
+;extension=php_sqlite.so
+;extension=php_sybase_ct.so
+;extension=php_tidy.so
+;extension=php_xmlrpc.so
+;extension=php_xsl.so
+;extension=php_zip.so
 
 ;;;;;;;;;;;;;;;;;;;
 ; Module Settings ;
